vmware_aria_operations_integration_sdk/mp_init.py

```diff
@@ -433,7 +457,8 @@ def build_project_structure(
         # create template requirements.txt
         requirements_file = os.path.join(path, "adapter_requirements.txt")
         with open(requirements_file, "w") as requirements:
-            requirements.write("psutil==5.9.4\n")
+            if template_style != NEW_ADAPTER_OPTION_KEY:
+                requirements.write("psutil==5.9.4\n")
```

Comment:
I would invert this to test for equality to the sample adapter -- if we ever have additional templates/options, the sample adapter is likely the only one that will require psutil